uint32 code
uint32 size
uint32 count
motion_udp/MessageData message_data
uint32 check_sum
